src/api/contract.ts

```diff
@@ -89,11 +91,18 @@ export function contract (context: ExtensionContext): ContractInterface {
     return await getConstructorInputFile(context, contractTitle)
   }
 
+  async function selectContract (
```

Comment:
We can have separate api `creating contract files` and `selecting contract file`

Reply:
![image](https://user-images.githubusercontent.com/72294477/228850595-8e46f31f-10f2-481a-ad23-537d319b1382.png)

we should have a proper check 
1. check if contract is compiled or not before moving further 
2. check if contract files are generated or not before contract selection 
3. before contract selection check if contract address exists for that or not , if not then return a warning